fulcrumapp/fulcrum-query-sql
# Changes to query.js to support created and updated coordinates when q…

# Solving missing coordinates 

Solving issue 2845 https://github.com/fulcrumapp/fulcrum/issues/2845 Please review this issue for more details on this problem and the necessity of these changes

## Problem

The record editor needs and expects the hidden created and updated coordinates for each record. Yet the current query object doesn't include these columns when it is building its query. 

## Solution

Include the missing columns in the query.  We are including created and updated lat, lon, altitude, and horizontal accuracy. 

## Testing

These were tested in the context of record editor. There are two ways to verify it.

1. Open record editor. Using a browser's dev tools, look for the query that populates the edit table. There should be a result with an object that lists fields and rows for the result. Confirm in the fields that the coordinates exist. Then confirm in the values that the index for the field has the appropriate values.

2. Create a form with a computed field that displays the created or updated coordinates, (this.updatedLatitude, for example ). Enter a record, and save it. Click on Edit. The computed field should have the correct data when the modal editor loads. 

# Converter changes

These are changes to fix linter complains about the number of spaces used in the indents of some lines.
